tests/testcases/mod.rs

```diff
@@ -321,7 +327,23 @@ pub fn bootstrap_smoke_test(
     };
     let mut build = std::process::Command::new(spin_binary());
     // Ensure `spin` is on the path
-    build.env("PATH", path).args(["build"]);
+    build.env("PATH", &path).args(["build"]);
+    build_env_vars.iter().for_each(|(key, value)| {
+        if key == "PATH" {
+            let mut custom_path = value.to_owned();
+            if value.starts_with('.') {
+                let current_dir = env.path();
+                custom_path = current_dir
```

Comment:
Consider pulling this out into a named function - it requires close reading of this code to see what's going on.  E.g.

```
let custom_path = env.absolutise_path(value);
```